mpmp/config.py

```diff
@@ -22,8 +22,8 @@
     'expression': expression_sample_info,
     'me_27k': me_27k_sample_info,
     'me_450k': me_450k_sample_info,
-    'rppa': rppa_sample_info,
-    'mut_sigs': mut_sigs_sample_info,
+    # 'rppa': rppa_sample_info,
+    # 'mut_sigs': mut_sigs_sample_info,
```

Comment:
Do you need to leave this code commented out here? I mention this just in case you don't.

Reply:
I do - for these experiments I want to use only the overlapping samples between the expression and methylation datasets (if I add the other data types, lots of samples get dropped), and commenting them out here was the easiest way to do that.

I should find a better way to deal with this in the future, though. I'll create an issue to remind myself.